src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.html

```diff
@@ -1,12 +1,57 @@
 <sofie-row class="mb-3">
   <form [formGroup]="actionTriggerForm" class="left-container">
-    <div [formGroupName]="'data'">
-      <sofie-text-input i18n-label label="global.label" [formControlName]="'label'"></sofie-text-input>
-      <sofie-keyboard-input [isRequired]="true" i18n-label label="action-triggers.shortcut.label" [formControlName]="'keys'" i18n-errorMessage errorMessage="action-triggers.shortcut.error">
+    <div [formGroupName]="'data'" style="position: relative;">
```

Comment:
Here and on line 14 as well you use inline styling which we really try to avoid, unless it can be done by our utility classes in: https://github.com/tv2/sofie-web-client/blob/staging/src/scss/_sofie-utility-classes.scss